catalog/gitops/README.md

```diff
@@ -1,6 +1,9 @@
-# CSR Git Ops Pipeline
+# GitOps package
```

Comment:
Suggest we keep CSR here since this blueprint only works with CSRs?

Reply:
It's using the name from the Kptfile, which ideally should match the name of the parent directory, according to kpt docs.

@morgante renamed the parent package to remove the csr detail and there was a discussion about CSR being the default impl (other impls would need an identifier in the name). So I named this child package accordingly.

If we want CSR in the name, it should be in the directory name and the kptfile.

Reply:
I don't agree that the name of the directory needs to match the name in the README. The README name should be more descriptive and full.

Please revert.

Reply:
> The directory rename was based on your suggestion. 

I did suggest shortening the old `csr-git-ops-pipeline` name, and you shortened it to `csr-gitops`, which is what's on main. 

After you had already merged that, I suggested using `gitops-csr` instead, to set a precedence to make others for -github and -gitlab and they would still sort next to each other alphabetically. But you shortened it to `gitops` instead for kpt-v1.

Perhaps you misunderstood my suggestion, but my recollection is that you explicitly did not want to use the -csr suffix because you wanted to make the GCP-native path the default.

I'll put the fact that it uses CSR back in the description, but I don't think it belongs in the title of the readme unless it's in the name of the package. This readme was manually modified, but I have a script to generate them directly from the kptfile, in which case it would be `# gitops package`.

Reply:
Yes, I chose to remove the `-csr` suffix so this can be the default.

The name for packages in the readme does not need (and probably should not) match the package name. In the directory name, we often prioritize brevity. the README title should prioritize descriptiveness.

Reply:
Just to re-emphasize the point, *most* of our packages use titles in the README which are different (more verbose) than the directory name. For example: https://github.com/GoogleCloudPlatform/blueprints/tree/main/catalog/bucket

There is absolutely no requirement for the titles to match and they should not. `google-cloud-storage-bucket` would be an excessively long directory name while `bucket page` would be a poor title for the package.

Reply:
> A human-readable description is not the same as a directory name.

I agree, and I think the name in the Kptfile should be human readable. If it were, then pulling it out to use for the readme title would be fine. But kpt docs and code say that they should be the same.

If we want them to be different, to facilitate readme generation, then we should ask the kpt team to change it. We can of course manually modify titles in the mean time, but then we can't use the readme generator for validation in pre-commit tests.

Reply:
go/blueprint-metadata calls for a different "name" (machine-oriented) vs "title" (human-oriented) which has not yet been implemented. I agree we should continue pushing for inclusion of `title` metadata.

For the time being, let's just manually modify titles. We will have to make the README generator be able to tolerate some discrepancies anyways.